standalone-metastore/metastore-server/pom.xml

```diff
@@ -25,6 +25,11 @@
     <standalone.metastore.path.to.root>..</standalone.metastore.path.to.root>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>org.liquibase</groupId>
+      <artifactId>liquibase-core</artifactId>
+      <version>4.19.0</version>
```

Comment:
Since this is a crucial dependency, I believe its best to keep the version - `4.19.0` as a separate version tag in standalone-metastore pom file and reference the version tag here.
Something like this - 
`<version>${liquibase.version}</version>`